src/shortcuts/cursorForward.js

```diff
@@ -1,12 +1,20 @@
+import React from 'react'
```

Comment:
Please add a newline before line comments. 

Make sure this convention is followed in all other files as well.

Reply:
Got it